### DISCUSSION TITLE
Select throttle with searchable prop enabled

### DISCUSSION THREAD

**Discussion by EgorPopovPP** [mantinedev/mantine#5931] [Q&A] [answered]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.6.2

### What package has an issue?

@mantine/core

### What framework do you use?

Vite

### In which browsers you can reproduce the issue?

Not applicable – issue is not related to the browser

### Describe the bug

Hi
I am using `<Select />` component along with `searchable` prop enabled.  I have encountered a problem with trotting when quickly scrolling through the list of options. Sometimes, the page just wouldn't respond

### If possible, include a link to a codesandbox with a minimal reproduction

https://codesandbox.io/p/sandbox/dazzling-easley-pz9cww?file=/src/App.tsx:468,1

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment, marked as answer]
- I do not see any issues in the sandbox, it works smooth
- If you need to optimize performance, use `limit` prop – https://mantine.dev/core/select/#large-data-sets
